azure-spring-boot/pom.xml

```diff
@@ -27,6 +27,7 @@
         <dependency>
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-validator</artifactId>
+            <version>6.0.4.Final</version>
```

Comment:
You shouldn't hardcode a version here. The version is still managed but the groupId has changed to `org.hibernate.validator`